scripts/measureAspireStartup.ps1

```
@@ -0,0 +1,106 @@
+        }
+
+        $env:BUDDY_DEBUG_PORT = $Port
+        $timer = [System.Diagnostics.Stopwatch]::StartNew()
+        try {
+            & $aspireCmd start --no-build --format Json --non-interactive --nologo |
+                Out-Null
+            if ($LASTEXITCODE -ne 0) {
+                throw "aspire start failed with exit code $LASTEXITCODE."
+            }
+
+            $dashboardSeconds = Wait-StartupPort $timer $dashboardPort $TimeoutSeconds
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Wait-StartupPort' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)